src/status_im/node/core.cljs

```diff
@@ -125,6 +125,13 @@
       :always
       (add-log-level log-level))))
 
+(defn get-node-config [db network]
```

Comment:
maybe call it `get-dummy-node-config`? or `get-verification-config` with a short comment that the verification won't work if the node isn't started.

Reply:
i actually restored a piece of code existed before last changes related to node management, but i don't mind to call it  `dummy-config` :)

Reply:
honestly i would prefer to rewrite `VerifyAccountPassword` so that it can be used without running node, i suppose that's doable. Not a part of this PR though

Reply:
changed